src/devices/Lp55231/MiscFlags.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace Iot.Device.Lp55231
+{
+    /// <summary>
+    /// Flags related to the REG_MISC register
+    /// </summary>
+    [Flags]
+    public enum MiscFlags : byte
+    {
+        /// <summary>
+        /// Use internal clock
+        /// </summary>
+        ClockSourceSelection = 0b00000001,
+
+        /// <summary>
+        /// Use external clock
+        /// </summary>
+        ExternalClockDetection = 0b00000010,
+
+        /// <summary>
+        /// Enable PWM cycle power save
+        /// </summary>
+        PWMCyclePowersaveEnable = 0b00000100,
```

Comment:
Pwm rather than PWM, we're using is like this everywhere. So it's for consistency.

Reply:
Done